Logger/LogFactory.cs

```diff
@@ -2,10 +2,27 @@
 {
     public class LogFactory
     {
-        public BaseLogger CreateLogger(string className)
+        private string FilePath { get; set; } = "";
```

Comment:
```suggestion
        private string _FilePath;
```
Auto properties are typically only used for public members. You can use a private variable for your file path string.